core/base/inc/TString.h

```diff
@@ -529,7 +544,7 @@ inline TString &TString::operator+=(Int_t i)
 inline TString &TString::operator+=(UInt_t i)
 { return operator+=((ULong_t) i); }
 
-inline TString &TString::operator+=(Double_t f)
+inline TString &TString::operator+=(LongDouble_t f)
```

Comment:
The `%.17g` below is the wrong format spec for `long double`. It should become `%.17Lg`.